### DISCUSSION TITLE
如何主动更新Setter的值

### DISCUSSION THREAD

**Discussion by runfi** [alibaba/designable#358] [Q&A]
背景：
1、在designable中，自定义了新的Setter组件。
2、希望异步获取某些内容传入Setter
请问：
1、如何传参进入Setter
1-1：因为定义Setter是在createBehavior的designerProps定义的，好像无法像组件那样传入参数？
2、如何主动更新Stter的value，

**Comment by lijie1024512** [comment]
resource的componentprops可以
